src/WCSimRootOptions.cc

```diff
@@ -25,7 +25,7 @@ WCSimRootOptions::~WCSimRootOptions()
 }
 
 //______________________________________________________________________________
-void WCSimRootOptions::Print()
+void WCSimRootOptions::Print(Option_t *option) const
```

Comment:
Can you change this to `(Option_t *)` or `(Option_t * /*option*/)`, to get rid of a compiler warning? :wink: